share/elections/2023-01/ewoud.md

```diff
@@ -0,0 +1,25 @@
+# Ewoud Kohl van Wijngaarden
+
+I, [Tim Meusel](https://github.com/bastelfreak), nominate Ewoud as a PMC
+candidate.
+
+Ewoud is awesome! That's it. That should be enough as an introduction.
+
+He is around in the Vox Pupuli community since 2016-02-22. I'm doing Puppet
+stuff since almost 10 years and he's the person I ask when I am stuck. Ewoud
+is a recognized expert in the foreman community and in Vox Pupuli. His PR
```

Comment:
```suggestion
is a recognized expert in the Foreman community and in Vox Pupuli. His PR
```

Reply:
because of the domain `https://theforeman.org/` I'm also never sure if the official project name is `Foreman` or `The Foreman` or `theforeman`.